pom.xml

```
@@ -222,12 +222,12 @@
 			<dependency>
 				<groupId>mysql</groupId>
 				<artifactId>mysql-connector-java</artifactId>
-				<version>8.0.19</version>
+				<version>8.0.26</version>
```

```suggestion
				<version>8.0.19</version>
```

Lets not update MySQL for now since we know that there were some issues with versions after 8.0.19

---

Reverted the update.